lib/tasks/eyl_eligible_schools.rake

```diff
@@ -0,0 +1,35 @@
+require "csv"
+
+namespace :eyl_eligible_schools do
+  desc "Make schools eligible for NPQEYL courses"
+  task :update, %i[file_name] => :environment do |_t, args|
+    updated_records = 0
+    update_errors = 0
+    file_name = args.file_name
+
+    raise "File not found: #{file_name}" unless File.exist?(file_name)
```

Comment:
Does the file need to be included in the repo?

Reply:
The file is included